src/Transposit.ts

```diff
@@ -92,22 +94,21 @@ export class Transposit {
 
     const codeVerifier = popCodeVerifier();
 
-    const response = await fetch(this.uri(`/login/authorize/token`), {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/x-www-form-urlencoded",
+    const tokenResponse = await trfetch<TokenResponse>(
+      this.uri(`/login/authorize/token`),
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
just curious, why use form-urlencoded instead of json?

Reply:
This is the recommendation from the OAuth RFC.